jobbergate-api/jobbergate_api/apps/services.py

```diff
@@ -585,15 +575,14 @@ async def _get_file_data_from_url(self, file_url: AnyUrl) -> io.BytesIO:
                     raise_exc_class=ServiceError,
                     raise_kwargs=dict(status_code=status.HTTP_400_BAD_REQUEST),
                 )
-
-                session = Session()
-                async with session.resource("s3", url_string) as s3:
-                    bucket = await s3.Bucket(bucket_name)
-                    s3_obj = await bucket.Object(key)
-                    await s3_obj.download_fileobj(file_obj)
+                file_url_string = f"https://{bucket_name}.amazonaws.com{key}"
+            case "http" | "https":
+                file_url_string = file_url.unicode_string()
             case _:
                 raise ServiceError(f"Unsupported protocol to get file data by url for {file_url}")
 
+        response = httpx.get(file_url_string)
+        file_obj.write(response.content)
```

Comment:
What do you think in making some assertion here to check if the file can be downloaded?

Reply:
Added error handling for the `GET` request in ed5878d.